tests/behat/features/bootstrap/AlbumContext.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Drupal\social\Behat;
+
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Drupal\Component\Plugin\Exception\PluginNotFoundException;
+use Drupal\DrupalExtension\Context\DrupalContext;
+use Drupal\DrupalExtension\Context\MinkContext;
+use Drupal\group\Entity\Group;
+use Drupal\node\Entity\Node;
```

Comment:
Drupal\group\Entity\Group and Drupal\node\Entity\Node are not used. Please remove them.